src/test/regress/sql/multi_router_planner.sql

```diff
@@ -929,7 +929,7 @@ UPDATE pg_dist_shard SET shardmaxvalue = 100, shardminvalue=1 WHERE shardid = :s
 
 -- we execute the query within a function to consolidate the error messages
 -- between different executors
-CREATE FUNCTION raise_failed_execution_router(query text) RETURNS void AS $$
+CREATE OR REPLACE FUNCTION raise_failed_execution_router(query text) RETURNS void AS $$
```

Comment:
I guess this means it wasn't dropped at the end of the file. Let's do that instead, and preferably with a schema drop.

Reply:
yeah, fixed that as well